### PR TITLE
danielchalmers.SteamAccountSwitcher version 2.3.1

### PR DESCRIPTION
**Pull request by danielchalmers**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/61946)

### CODE REVIEW

**Comment by yao-msft** [comment]
A bunch of failures in Daas log, + @hackean-msft to see if there's any improvements could be made.

**Comment by danielchalmers** [comment]
> A bunch of failures in Daas log, + @hackean-msft to see if there's any improvements could be made.

It's a ClickOnce installer, is that supported?

**Comment by yao-msft** [comment]
> > A bunch of failures in Daas log, + @hackean-msft to see if there's any improvements could be made.
> 
> It's a ClickOnce installer, is that supported?

It is supported. Actually in this re-run, all failures are gone.

**Comment by yao-msft** [comment]
When trying to launch the app, SmartScreen blocked it.

File: manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1
+PackageLocale: en-US
+Publisher: Daniel Chalmers
+PublisherUrl: https://github.com/danielchalmers
+Author: Daniel Chalmers
+PackageName: SteamAccountSwitcher
```
Comment (ItzLevvie) [inline manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml:10]:
```suggestion
PackageName: Steam Account Switcher
```
This must match Control Panel.

File: manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1
```
Comment (ItzLevvie) [inline manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.3.1.0
```
This must match Control Panel.

File: manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1
```
Comment (ItzLevvie) [inline manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.installer.yaml:5]:
```suggestion
PackageVersion: 2.3.1.0
```
This must match Control Panel.

File: manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1
```
Comment (ItzLevvie) [inline manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.yaml:5]:
```suggestion
PackageVersion: 2.3.1.0
```
This must match Control Panel.

File: manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Publisher: Daniel Chalmers
+PublisherUrl: https://github.com/danielchalmers
+Author: Daniel Chalmers
+PackageName: SteamAccountSwitcher
+PackageUrl: https://danielchalmers.github.io/SteamAccountSwitcher
+License: MIT license
+LicenseUrl: https://github.com/danielchalmers/SteamAccountSwitcher/blob/master/LICENSE.md
+Copyright: © Daniel Chalmers
+ShortDescription: Quickly switch between your Steam accounts
+Moniker: SteamAccountSwitcher
```
Comment (ItzLevvie) [inline manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml:16]:
```suggestion
Moniker: steam-account-switcher
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**